p2p/src/types.rs

```diff
@@ -202,6 +202,9 @@ bitflags! {
 	/// Can provide a list of healthy peers
 	const PEER_LIST = 0b00000100;
 
+	const FAST_NODE = Capabilities::TXHASHSET_HIST.bits
```

Comment:
A bit of a nitpick but `FAST_SYNC_NODE` may be clearer. Hopefully all our nodes are fast.

Reply:
Okay 👍